apps/publisher/src/hooks/useCanCaptureStream.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from 'react';
+
+/**
+ * Determine if HTMLMediaElement.captureStream is supported on the current browser
+ */
+const useCanCaptureStream = () => {
+  const [canCaptureStream, setCanCaptureStream] = useState(false);
+
+  useEffect(() => {
+    const video = document.createElement('video') as HTMLVideoElement & {
```

Comment:
We should not create a `video` element here,  the caller should pass the video element as an argument.